### ISSUE TITLE
Attempted to teleport to a place that is restricted. (Error Code: 773)

### ISSUE THREAD

**Issue by nalhiff** [pizzaboxer/bloxstrap#233]
![image](https://github.com/pizzaboxer/bloxstrap/assets/36839431/45f27084-9bbb-4d03-a93e-8f0b663afeb4)

i tried to use multiple-game instances for dual login, its fine on the game lobby but when tried to teleport shows the issue. 

googling it leading to dual - login issues. 

i use [RAM](https://github.com/ic3w0lf22/Roblox-Account-Manager) for now. but would be amazing if i could just use it outta browser 

thanks for your efforts pizza

**Comment by pizzaboxer** [comment]
Nothing I can really do about that. Doesn't RAM just launch Bloxstrap anyway?

**Comment by adrardossa** [comment]
in RAM, we had to put place id and job id if we're going to join private server

its much more convenient and faster to use links just like bloxstrap, but the teleport issue is exist while on RAM, it doesnt.. not sure why

**Comment by pizzaboxer** [comment]
Isn't the only difference here just that you're copying the Job ID with Bloxstrap instead of copying it from the website?

**Comment by adrardossa** [comment]
nope, with bloxstrap i can join directly through links.. on the other hand using RAM would be copying place id on the links.

i like to playing co-op with randoms through private server, since i need to copying the place id which is slower and the other randoms usually faster than me and maxed out the slots. you know first come first serve kinda things

**Comment by pizzaboxer** [comment]
Okay, I think this is making a bit more sense. Do your alt accounts even have permission to join the private server independently without the need to generate a link?

**Comment by pizzaboxer** [comment]
Weird, I don't really know why RAM is treating Job ID joining as a teleport. Either way, it's hard to really call this a Bloxstrap issue since Bloxstrap is just giving you the Job ID, RAM is doing the actual launching for it.

**Comment by adrardossa** [comment]
ram was doing fine, i was hoping that i could do dual login without ram (just by bloxstrap multi instances) since its much more convenient which i dont need to copy job id and place id.. its not working when doing the teleport things.

**Comment by pizzaboxer** [comment]
Yeah, there's not really much I can do about that since it's purely a permission error with however you're doing game joining. Sorry.

**Comment by pizzaboxer** [comment]
> googling it leading to dual - login issues.

I just realized, you said you're trying to launch Roblox from your browser, right?

**Comment by adrardossa** [comment]
> > googling it leading to dual - login issues.
> 
> I just realized, you said you're trying to launch Roblox from your browser, right?

yes, by bloxstrap

**Comment by pizzaboxer** [comment]
How do you enter the Job ID to launch with?

**Comment by adrardossa** [comment]
just put the private server llinks on it

**Comment by pizzaboxer** [comment]
Oh, that makes a lot more sense. Yeah, those won't launch corresponding to the account that you're currently signed into your browser. Those will just launch with to whatever account is signed into the Roblox app, and so you get a duplicate login problem. RAM launches with account data, so it's not a problem there.

**Comment by adrardossa** [comment]
im using different browser for different account, the issue still persist

**Comment by adrardossa** [comment]
the private server on roblox game
![image](https://github.com/pizzaboxer/bloxstrap/assets/36839431/e70155b9-227d-46e2-93ee-1c2af1b0a898)

**Comment by pizzaboxer** [comment]
Yeah, but what's the link that you paste the Job ID into?
